### PR TITLE
IBX-5973: Virtual Products checkout added in Dev-doc

### PR DESCRIPTION
**Pull request by julitafalcondusza**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | (https://issues.ibexa.co/browse/IBX-5973)
| Versions      | master
| Edition       | Commerce

Content about Virtual Products added in:
- 'Checkout' page,
- 'Products' page.

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date
- [x] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/commerce/checkout/checkout.md [hunk 1]
```diff
@@ -47,3 +47,13 @@ an individual.
 - Individuals will be able to enter both addresses at checkout
 
 For more information about shipping and billing addresses, see [Configure checkout](configure_checkout.md#shipping-and-billing-address-field-format-configuration).
+
+## Virtual Products checkout
+
+Virtual product is a special type of a [Product](products.md). Virtual products are a non-tangible items such as memberships, services, warranties.
```
Comment (radzyu) [inline docs/commerce/checkout/checkout.md:53]: without "a" as you used items that is plural

File: docs/commerce/checkout/checkout.md [hunk 2]
```diff
@@ -47,3 +47,13 @@ an individual.
...
 
 For more information about shipping and billing addresses, see [Configure checkout](configure_checkout.md#shipping-and-billing-address-field-format-configuration).
+
+## Virtual Products checkout
+
+Virtual product is a special type of a [Product](products.md). Virtual products are a non-tangible items such as memberships, services, warranties. 
+They can be sold individually, or as part of a product bundle.
+
+Virtual products don’t require shipment when they are purchased individually.
+While purchasing virtual product, you only have to fill in Billing address.
```
Comment (radzyu) [inline docs/commerce/checkout/checkout.md:57]: and select relevant payment method.

File: docs/pim/products.md [hunk 3]
```diff
@@ -20,6 +20,14 @@ A product type can be, for example, a sofa or a keyboard.
 Product types, like Content Types, define the global properties of products and Fields a product consists of.
 A product type also defines the attributes that all products of this type can have.
 
+You can choose between two available types: `Physical` and `Virtual`:
+
+- `Physical` - a tangible products with assigned stock.
```
Comment (radzyu) [inline docs/pim/products.md:25]: do we need "a" here?

**Review by radzyu — CHANGES_REQUESTED** [review]
Should we also mention somewhere how we deal with payment method filtering? @ciastektk ?

**Review by radzyu — APPROVED** [review]

**Review by ciastektk — APPROVED** [review]

File: docs/commerce/checkout/checkout.md [hunk 4]
```diff
@@ -47,3 +47,13 @@ an individual.
 - Individuals will be able to enter both addresses at checkout
 
 For more information about shipping and billing addresses, see [Configure checkout](configure_checkout.md#shipping-and-billing-address-field-format-configuration).
+
+## Virtual Products checkout
+
+Virtual product is a special type of a [Product](products.md). Virtual products are non-tangible items such as memberships, services, warranties.
```
Comment (MagdalenaZuba) [inline docs/commerce/checkout/checkout.md:53]:
```suggestion
Virtual product is a special type of a [product](products.md). Virtual products are non-tangible items such as memberships, services, warranties. 
```

File: docs/commerce/checkout/checkout.md [hunk 5]
```diff
@@ -47,3 +47,13 @@ an individual.
...
 
 For more information about shipping and billing addresses, see [Configure checkout](configure_checkout.md#shipping-and-billing-address-field-format-configuration).
+
+## Virtual Products checkout
+
+Virtual product is a special type of a [Product](products.md). Virtual products are non-tangible items such as memberships, services, warranties. 
+They can be sold individually, or as part of a product bundle.
+
+Virtual products don’t require shipment when they are purchased individually.
+While purchasing virtual product, you only have to fill in Billing address and select relevant payment method.
```
Comment (MagdalenaZuba) [inline docs/commerce/checkout/checkout.md:57]:
```suggestion
While purchasing virtual product, you only have to fill in the billing address and select relevant payment method. 
```

**Review by MagdalenaZuba — APPROVED** [review]

**Pull request merged**